demaceo/overlook
# Feature/bookings

### What does  this PR do?
Allows a guest or manager to book &/or delete a room. 

### How should this be manually tested?
By logging in with the users credentials, click on a date & then click on a room available. you will then see a message pop up confirming the newly booked room.

### What are the relevant tickets?
closes #15 #14
